prow/git/v2/adapter.go

```diff
@@ -44,12 +44,21 @@ type clientFactoryAdapter struct {
 
 // ClientFromDir creates a client that operates on a repo that has already
```

Comment:
Move this comment back to its original location + add a comment for this new function